src/transformers/models/ctrl/modeling_tf_ctrl.py

```diff
@@ -737,9 +750,6 @@ def __init__(self, config, *inputs, **kwargs):
         )
         self.transformer = TFCTRLMainLayer(config, name="transformer")
 
-    def get_output_embeddings(self):
```

Comment:
`TFCTRLForSequenceClassification` incorrectly had a `get_output_embeddings()`, and got the test overridden -- sequence classification models do not have a LM Head (and therefore no output embeddings)

I also removed the overridden test

Reply:
Not so fast! :D 

Even if it's not correct to have this method - it's one that belongs to a public model class. We should have a deprecation cycle first warning users that it will be deleted in a few versions' time